docs/getting-started.stories.mdx

```diff
@@ -174,23 +176,31 @@ import ReactDOM from "react-dom";
 
 #### Theming
 
-Carbon uses the `ThemeProvider` from the [styled-components library](https://styled-components.com/docs/advanced#theming) to supply theme styles to your components. Themes are define within the Carbon library.
+Carbon provides the <LinkTo kind="Carbon Provider" story="default">Carbon Provider</LinkTo> component that uses the `ThemeProvider` from the [styled-components library](https://styled-components.com/docs/advanced#theming) to supply theme styles to your components. Themes are define within the Carbon library. `mintTheme` is provided by default.
 
 ```js
-import { ThemeProvider } from "styled-components";
-import mintTheme from "carbon-react/lib/style/themes/mint";
+import { CarbonProvider } from "styled-components";
```

Comment:
This import path isn't correct

Reply:
and it should be a default import rather than named

Reply:
changed, thanks